src/wp-includes/class-wp-theme.php

```diff
@@ -1893,6 +1903,10 @@ public function get_block_patterns() {
 		foreach ( $files as $file ) {
 			$pattern = get_file_data( $file, $default_headers );
 
+			if ( empty( $pattern['slug'] ) && empty( $pattern['title'] ) ) {
```

Comment:
I think this makes sense to add here. The downside may be that folks forget to add the header and have their patterns silently fail to load.

Reply:
You are right. I'm going to remove this code.